packages/server/utils/config.ts

```diff
@@ -1,5 +1,5 @@
 import dotenv from "dotenv";
-dotenv.config({ path: "../../.env"});
```

Comment:
Is there a reason we need this `dotenv.config()` here when it's already being called in `packages/server/server.ts`?

Reply:
I think any file that wants to directly access `process.env` needs the import, but you may be right that it could be better to DRY the code by doing all the `process.env` calls in a single file and then exporting them from that file so other files can consume as necessary

Reply:
Good call. That is much cleaner.